metaapp/src/main/java/com/sourcepointmeta/metaapp/adapters/PropertyListAdapter.java

```diff
@@ -71,6 +72,7 @@ public void onBindViewHolder(@NonNull PropertyViewHolder viewHolder, int positio
         viewHolder.propertyNameTextView.setText(name);
         viewHolder.propertyAccountID.setText("Account ID : " + accountID);
         viewHolder.propertyCampaign.setText("Campaign : " + campaign);
+        viewHolder.propertyType.setText("PropertyType : "+ propertyType);
```

Comment:
I believe we could rename `propertyType` to `messageType`. The property has no type, it's the message that can be web or native.

Reply:
@andresilveirah Yes we can rename it , I would change it to messageType.